odoo/tools/translate.py

```diff
@@ -1473,10 +1473,10 @@ def load_language(cr, lang):
 def get_po_paths(module_name: str, lang: str):
     lang_base = lang.split('_')[0]
     langs = [lang_base, lang]  # the latter will overwrite the former
-    if lang_base == "es" and lang != "es_MX":
-        # force es_MX as fallback language for the different spanish
-        # es_MX is more actively translated and closer to many languages
-        langs = [lang_base, "es_MX", lang]
+    if lang_base == "es" and lang != "es_419":
+        # force es_419 as fallback language for the spanish variations
+        lang_base = "es_419"
```

Comment:
`langs = [lang_base, 'es_419', lang]`

Reply:
No, the idea would be to have two totally different languages and no longer rely on es for es_419 (we will populare that will es_MX content as it's already fully translated)

Reply:
I mean changing `lang_base = "es_419"` is useless here because the variable is not used after

Reply:
Now `es_ES` will also fallback to `en_419`
and file es.po will never been used

Is this the expected behavior?